example/postgres-image/run

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```

Comment:
Do you need this? Use of docker-compose with build as target would be enough? 
If you need this, add .sh to the filename, add the execution bit.

Reply:
C&P product from postgres-grpc connector but it shouldn't be needed